src/main/java/net/dries007/tfc/objects/inventory/slot/SlotQuernHandstone.java

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
None of this is necessary, all the functionality should be able to be handled by `SlotCallback`